msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SSPRTest.kt

```diff
@@ -56,20 +57,20 @@ class SSPRTest : NativeAuthPublicClientApplicationAbstractTest() {
      * Verify email with email OTP first and then reset password.
      * (hero scenario 8 & 17, use case 3.1.1, Test case 46)
      */
-    @Ignore("Fetching OTP code is unstable")
     @Test
     fun testSSPRSuccess() = runBlocking {
         var result: ResetPasswordStartResult
-        var otp: String
```

Comment:
[Copied from Silviu: nit: Are these changes needed? Wouldn't they work even if the otp was declared outside the block?](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2189#discussion_r1820575174)